### ISSUE TITLE
i18N_subsites and feed generation

### ISSUE THREAD

**Issue by arteteco** [getpelican/pelican-plugins#1033]
Hello everyone,

I don't seem to be able to find it, is there a pre-thought way of having feeds for different language in different atom files? I don't think readers of one language care about the translations of the same article...

I can work around it by using categories, but it's a bit of an hack.

Thanks!

**Comment by jorgesumle** [comment]
> is there a pre-thought way of having feeds for different language in different atom files?

Yes, [have a look at `TRANSLATION_FEED_RSS`](http://docs.getpelican.com/en/stable/settings.html#translations).

**Comment by haplo** [comment]
> > is there a pre-thought way of having feeds for different language in different atom files?
> 
> Yes, [have a look at `TRANSLATION_FEED_RSS`](http://docs.getpelican.com/en/stable/settings.html#translations).

But `TRANSLATION_FEED_RSS` doesn't generate per-category, per-tag or per-author feeds for each subsite, does it? In other words, I believe it is equivalent to the [FEED_ATOM](http://docs.getpelican.com/en/stable/settings.html#feed-settings) setting, am I wrong?

It would be great to have support for all possible feeds on each i18n subsite, so readers of each site have the same feeds available.
